src/main/java/ru/ayaz/Comment.java

```diff
@@ -0,0 +1,23 @@
+package ru.ayaz;
+
+import java.util.Calendar;
+import java.util.Date;
+
+public class Comment {
```

Comment:
You eventually will have to figure how to compare comments with each other (you are deleting comments and searching for them, but do you know how List<Comment>.contains(comment) behaves ? :) )

=== Spoiler ===
When you issue a comparison operation between comments through List<Comment>.contains(comment) or the like then "equals" method kicks in. Its the result of "equals" that decides whether or not two objects are equal.
And, as I see, you didn't redefine "equals" method. Which means you are using "equals" method inherited from Object class. Which means you are comparing your comments via "==" which means you are comparing their reference addresses. Which means that each newly created object is different from the previous one (well, there might be a situation when its not true, but its irrelevant) regardless of the content of its member fields.